schema/va-spec/base/pathogenicity-evidenceline-profile-source.yaml

```diff
@@ -0,0 +1,100 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/base/pathogenicity-evidenceline-profile-source.yaml"
+title: Variant Pathogenicity EvidenceLine Standard Profile
+strict: true
+
+imports:
+  domain-entities: ./domain-entities-source.yaml
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  # Pathogenicity Evidence Line
+  PathogenicityEvidenceLine:    
+    inherits: gks-core:EvidenceLine
+    maturity: trial use
+    type: object
+    description: >-
+      An Evidence Line that describes the strength and direction of support provided by one or more 
+      evidence items for or against the pathogenicity of a variant for a particular disease. 
+    properties:
+      type:
+        extends: type
+        const: "pathogenicityEvidenceLine"
+        default: "PathogenicityEvidenceLine"
+        description: MUST be "PathogenicityEvidenceLine".
+      hasEvidenceItems:
+        extends: hasEvidenceItems
+        type: array
+        ordered: false
+        items:
+          $ref: "/ga4gh/schema/gks-core/1.x/json/InformationEntity"
+        description: >-
+          An Information Entity (e.g. a Functional Impact Study Statement) that was assessed as
+          evidence in determining the possible pathogenicity of a variant. 
+      directionOfEvidenceProvided:
+        extends: directionOfEvidenceProvided
+        description: >-
+          The direction of support that the Evidence Line is determined to provide for its target
+          Variant Pathogenicity Proposition (i.e. does the evidence line support or dispute the
+          pathogenicity of the assessed variant, or remain neutral)
+      strengthOfEvidenceProvided:           # TO DO: Ensure that ten enum below covers terms needed for initial adopters. Merge https://github.com/ga4gh/gks-common/pull/78 which updates the core IM to allow this property to take a string. Think about if we need the PS/BS codes here - as this bakes in direction, and is duplicative as the 'specifiedBy' attribute reports the ACMG criterion applied. 
+        extends: strengthOfEvidenceProvided
+        type: string
+        enum:
+          - PS3_Strong
+          - PS3_Moderate
+          - PS3_Supporting
+          - BS3_Strong
+          - BS3_Moderate
+          - BS3_Supporting
```

Comment:
This needs to be changed for several reasons:
1. If it truly meant to hold the specific MappableConcepts for each of the ACMG-2015 codes then it should be moved into the specialized `VariantPathogenicityEvidenceLine` in the `ACMG-2015` folder/namespace and be made more comprehensive. 
2. The `EvidenceLine.strengthOfEvidenceProvided` attribute in `gks-core` is of type `MappableConcept` at this time. So we either need to change that to a string or change this to a `MappableConcept` or do something completely different that will align the datatypes in both?  (possibly creating a specialized class or start making a plan for `ValueSet`s)

Reply:
I agree with these points. I might add that we may be better off using a shared, generic, concrete EvidenceLine class that contains all of the attributes seen in the (Variant)PathogenicityEvidenceLine, which is constrained within the ACMG-2015 namespace. Unless we think there are entire properties that are specific to all (Variant)PathogenicityEvidenceLines but not other evidence lines (e.g. AMP/ASCO/CAP evidence lines).

Reply:
@mbrush and @larrybabb will work through the plan to add an `evidenceOutcome` property on `EvidenceLine` which will summarize the met or not met criteria derived from the methods.